srcDLL/op2ext.cpp

```diff
@@ -119,3 +119,22 @@ OP2EXT_API size_t GetLoadedModuleCount()
 {
 	return iniModuleLoader.Count() + consoleModLoader.Count();
 }
+
+OP2EXT_API size_t GetLoadedModuleName(size_t moduleIndex, char* buffer, size_t bufferSize)
+{
+	// Ini modules are indexed first and console module is last
+
+	std::string moduleName;
+
+	if (moduleIndex < iniModuleLoader.Count()) {
+		moduleName = iniModuleLoader.GetModuleName(moduleIndex);
+	}
+	else if (moduleIndex < GetLoadedModuleCount()) {
```

Comment:
This check is mildly unintuitive as I was half expecting it to read `consoleModuleLoader.Count()` rather than `GetLoadedModuleCount()`. I suppose I was also expecting some kind of re-indexing (such as `moduleIndex - iniModuleLoader.Count()`), though there is currently no point.

I think I'm going to say no change required. I can't think of any better suggestions here.